server/chat-plugins/github.ts

```diff
@@ -0,0 +1,266 @@
+/**
+ * Chat plugin to view GitHub events in a chatroom.
+ * By Mia, with design / html from xfix's original bot, https://github.com/xfix/GitHub-Bot-Legacy/
+ * @author mia-pi-git
+ */
+
+import {FS, Utils} from '../../lib';
+
+const STAFF_REPOS = ['pokemon-showdown', 'pokemon-showdown-client', 'Pokemon-Showdown-Dex'];
```

Comment:
The Dex isn’t on the smogon org

Reply:
Said this in DMs, but repeating in public: as long as Zarel sets up the hook on the Dex repo, the org doesn't really matter.